control-plane-operator/hostedclusterconfigoperator/controllers/resources/rbac/reconcile.go

```diff
@@ -631,3 +631,89 @@ func ReconcileUserOAuthClusterRole(r *rbacv1.ClusterRole) error {
 	}
 	return nil
 }
+
+func ReconcileAzureDiskCSIDriverNodeServiceAccountRole(r *rbacv1.ClusterRole) error {
```

Comment:
These are all `ClusterRole` and `ClusterRoleBinding`.  Can we change the function names to reflect this?